src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -554,9 +553,12 @@ private static async Task<(ITypeSymbol, RefKind)> GetArgumentTypeAndRefKindAsync
             //  `void Goo(BaseType baseType)`.  
             //
             // We want this simple case to match.
-            var conversion = compilation.ClassifyCommonConversion(argumentTypeInfo.Type, parameterType);
-            if (conversion.IsImplicit)
-                return true;
+            if (argumentTypeInfo.Type != null)
```

Comment:
❔ Is this not an intended precondition of the method? If it is, we could just add the following at the start of the (private) method:

```
Contract.ThrowIfNull(argumentTypeInfo.Type);
```

Reply:
Looking at call sites, it looks more complicated (i.e. null is allowed for this property if the conditions above produce an early return). We could still add a `Contract.ThrowIfNull`, but it would need to be at this location in the method.